package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@adia-technology/policies-consent-widget",
-  "version": "1.1.5",
+  "version": "1.1.6",
```

Comment:
New functionality is rather considered minor version bump, but ok.

Reply:
change from 1.1.5 to 1.1.6 is not considered as minor?
or you referrning to commit name?